src/components/_progress.scss

```diff
@@ -0,0 +1,52 @@
+.progress {
+  @include border-style(5);
+  border: 2px solid $primary;
+  box-shadow: 2px 8px 4px -6px hsla(0%, 0%, 0%, 0.3);
+  height: 1.2rem;
+  overflow: hidden;
+  width: 100%;
+
+  .bar {
+    @include border-style(5);
+    @include transition;
+    background-color: $primary-light;
+    border-right: 2px solid $primary;
+    display: flex;
+    flex-direction: column;
+    font-size: 0.6rem;
+    height: 100%;
+    justify-content: center;
+    text-align: center;
+    width: 0%;
+
+    @mixin striped-background($base-color) {
```

Comment:
I'd put this `@mixin` either at the top of this file or move it into the config altogether, but nesting that into the selectors is a little bit messy, imho

Reply:
Maybe it would also make sense to clean up the config in the process. I think it would be helpful if the `_config.scss` would only contain the variables. So only this part: https://github.com/papercss/papercss/blob/b6db8fad0659f45e04cf45f15b7ba96409c5a2bb/src/core/_config.scss#L1-L80

I'd move all the mixins below that into a new `_mixins.scss` file in `src/config` and add your `striped-background` mixin there. That way you can move the mixin into the right place and we also keep the config clean and minimal.

That would make it easier in the future to, for example, add themes to PaperCSS as the only thing you would need to edit to create a new theme would be the `$vars` inside the config. And since all those are tagged with `!default` you could do this without even changing a single character inside the config directly.